src/components/map-marker/map-marker.tsx

```diff
@@ -62,16 +69,38 @@ export class MapMarker implements GxComponent {
    */
   @Event() gxMapMarkerDeleted: EventEmitter;
 
+  private setPopup() {
+    const popupHtml = this.element.querySelector("[class='popupHtml']");
```

Comment:
Nothing big here, just wanted to mention that `popupHtml` doesn't really describe the content of the variable, right? I mean, it's a reference to a DOM element, not HTML.
It confused me at first a little.

Reply:
```popupHtml``` is about the HTML that will be inside in popup of marker. I need a way to check if there is any content inside ```<gx-map-marker>```, so, I use the container ```<div class="popupHtml">``` to check out the content. Perhaps I should rename it to _"popupContent"_ or _"popupHTMLContainer"_ ? Wht do u suggest?

Reply:
If `popupHtml` references an element, a better name could be `popupEl` or `popupContainerEl` if you want to be more specific about what the element is for.

Reply:
Mmm... I must do reference (using a variable) to the popup data container ```<div>```. To that ```<div>```, I will rename to its class as _"popupDataContainer"_, and the variable that refers to that element I will call it as _"popupContainerEl"_ . wht do you think about it?

Reply:
+1 with the variable name 👍 
As for the class name, I welcome the renaming. However, we are using dash separated, lower case names for classes, like `popup-data-container`, so please reconsider the new name.

Reply:
fixed on f956dec commit